src/nimble.nim

```diff
@@ -1857,17 +1857,17 @@ proc setupNimbleConfig(options: Options) =
     configFileVersion = 2
     sectionEnd = "# end Nimble config"
     sectionStart = "# begin Nimble config"
-    configFileHeader = &"# begin Nimble config (version {configFileVersion})\n"
+    configFileHeader = &"# begin Nimble config (version {configFileVersion})"
     configFileContentNoLock = fmt"""
 {configFileHeader}
-when system.fileExists("{nimblePathsFileName}"):
+when withDir(thisDir(), system.fileExists("{nimblePathsFileName}")):
   include "{nimblePathsFileName}"
 {sectionEnd}
 """
     configFileContentWithLock = fmt"""
 {configFileHeader}
 --noNimblePath
```

Comment:
We've agreed that this should be conditional on the presence of the lockfile.

Reply:
The reason for using lock file is that you can write the following...

```
nimble --lock-file=differentLockFileName.lock setup
```

... and it will still do the right thing. I don't have a strong preference for the approach here - both seem to be good enough.